lib/connectors/memory.js

```diff
@@ -823,7 +824,7 @@ Memory.prototype.replaceOrCreate = function(model, data, options, callback) {
     // Calling _createSync to update the collection in a sync way and
     // to guarantee to create it in the same turn of even loop
     return self._createSync(model, data, function(err, id) {
-      if (err) return process.nextTick(function() { cb(err); });
+      if (err) return process.nextTick(function() { callback(err); });
```

Comment:
Why not cb? I still prefer `cb` over `callback` unless we've decided to do this all over the place.